bonsai-core/src/test/scala/com/stripe/bonsai/GenericTree.scala

```diff
@@ -28,12 +28,10 @@ object GenericTree {
   }
 
   implicit def arbitraryTree[A: Arbitrary]: Arbitrary[GenericTree[A]] =
-    Arbitrary(genTree(arbitrary[A], 5, 12))
+    Arbitrary(genTree(arbitrary[A], 3, 7))
```

Comment:
Just running too slow (and producing unreadably large trees when tests failed for other reasons). I can change it back if you want.